Linux/jenkins-node/ansible/jenkins-agent-production.yml

```diff
@@ -2,16 +2,22 @@
   hosts: all
   vars:
     package_updates_reboot: true
+    deploy_type: production
+    jenkins_url: https://builds.mantidproject.org
     pip_install_packages:
       - name: docker
 
   roles:
     - role: dannixon.system.package_updates
+      tags: "initial-setup"
     - role: dannixon.system.interactive_users
+      tags: "initial-setup"      
     - role: geerlingguy.pip
       become: yes
+      tags: "initial-setup"      
     - role: geerlingguy.docker
       become: yes
+      tags: "initial-setup"      
```

Comment:
Can you add documentation for when might be a good time to use the `initial-setup` and `agent` tags?